### PR TITLE
Update Release Notes for oneDPL 2022.2.0

### PR DESCRIPTION
**Pull request by ValentinaKats**
Signed-off-by: Valentina Kats valentina.kats@intel.com

### CODE REVIEW

File: documentation/release_notes.rst [hunk 1]
```diff
@@ -8,6 +8,49 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
...
+- Fixed an error in ``reduce_by_segment`` algorithm when a non-commutative predicate is used.
+- Fixed ``sort`` and ``stable_sort`` algorithms when sorting integral types with width more than 32 bits
+  and using radix sort approach.
+- Fixed an error for some compilers attempting to use openMP or SYCL backend without full compiler support.
+
+Known Issues and Limitations
+----------------------------
+New in This Release
+^^^^^^^^^^^^^^^^^^^
+- Incorrect results may be produced with in-place scans using ``unseq`` and ``unseq_par`` policies on
```
Comment (akukanov) [inline documentation/release_notes.rst:40]:
```suggestion
- Incorrect results may be produced with in-place scans using ``unseq`` and ``par_unseq`` policies on
```

File: documentation/release_notes.rst [hunk 2]
```diff
@@ -8,6 +8,49 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
...
+
+Fixed Issues
+------------
+- Updated minimum CMake version to 3.11 on Linux and 3.20 on Windows.
+- Added new CMake package ``oneDPLIntelLLVMConfig.cmake`` to resolve issues using CMake 3.20+ on Windows for icx and icx-cl.
+- Fixed an error in the ``sort`` and ``stable_sort`` algorithms when performing a descending sort
+  on signed numeric types with negative values.
+- Fixed an error in ``reduce_by_segment`` algorithm when a non-commutative predicate is used.
+- Fixed ``sort`` and ``stable_sort`` algorithms when sorting integral types with width more than 32 bits
+  and using radix sort approach.
```
Comment (akukanov) [inline documentation/release_notes.rst:33]:
```suggestion
- Fixed an error in ``sort`` and ``stable_sort`` algorithms for integral types wider than 4 bytes.
```

File: documentation/release_notes.rst [hunk 3]
```diff
@@ -8,6 +8,49 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
...
+  ``is_partitioned``, and ``lexicographical_compare`` algorithms with DPC++ execution policies.
+- Improved performance of the ``reduce_by_segment``, ``inclusive_scan_by_segment``, and
+  ``exclusive_scan_by_segment`` algorithms for binary operators with known identities
+  when using DPC++ execution policies.
+- Added ``value_type`` to all views in ``oneapi::dpl::__ranges``. 
+- Added projection operator for ``oneapi::dpl::experimental::ranges::sort`` which is applied to the keys prior to sorting.
+
+Fixed Issues
+------------
+- Updated minimum CMake version to 3.11 on Linux and 3.20 on Windows.
```
Comment (akukanov) [inline documentation/release_notes.rst:27]:
```suggestion
- The minimally required CMake version is raised to 3.11 on Linux and 3.20 on Windows.
```

File: documentation/release_notes.rst [hunk 4]
```diff
@@ -8,6 +8,49 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.2.0
+===============
+
+New Features
+------------
+- Added ``sort_by_key`` algorithm.
```
Comment (akukanov) [inline documentation/release_notes.rst:16]:
```suggestion
- Added ``sort_by_key`` algorithm for key-value sorting.
```

Note also that this new algorithm needs to be documented, as well as proposed for adding to the specification.

File: documentation/release_notes.rst [hunk 5]
```diff
@@ -8,6 +8,49 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
...
+New Features
+------------
+- Added ``sort_by_key`` algorithm. 
+- Improved performance of the ``reduce``, ``min_element``, ``max_element``, ``minmax_element``,
+  ``is_partitioned``, and ``lexicographical_compare`` algorithms with DPC++ execution policies.
+- Improved performance of the ``reduce_by_segment``, ``inclusive_scan_by_segment``, and
+  ``exclusive_scan_by_segment`` algorithms for binary operators with known identities
+  when using DPC++ execution policies.
+- Added ``value_type`` to all views in ``oneapi::dpl::__ranges``. 
+- Added projection operator for ``oneapi::dpl::experimental::ranges::sort`` which is applied to the keys prior to sorting.
```
Comment (akukanov) [inline documentation/release_notes.rst:23]:
```suggestion
- Extended ``oneapi::dpl::experimental::ranges::sort`` to support projections applied to the range elements prior to comparison.
```

File: documentation/release_notes.rst (hunk 4 above)
Comment (MikeDvorskiy) [reply documentation/release_notes.rst]: For the doc - agree.
But for the spec  - there is no any mention about algo names.

File: documentation/release_notes.rst [hunk 6]
```diff
@@ -8,6 +8,48 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
...
+
+Fixed Issues
+------------
+- The minimally required CMake version is raised to 3.11 on Linux and 3.20 on Windows.
+- Added new CMake package ``oneDPLIntelLLVMConfig.cmake`` to resolve issues using CMake 3.20+ on Windows for icx and icx-cl.
+- Fixed an error in the ``sort`` and ``stable_sort`` algorithms when performing a descending sort
+  on signed numeric types with negative values.
+- Fixed an error in ``reduce_by_segment`` algorithm when a non-commutative predicate is used.
+- Fixed an error in ``sort`` and ``stable_sort`` algorithms for integral types wider than 4 bytes.
+- Fixed an error for some compilers attempting to use openMP or SYCL backend without full compiler support.
```
[the remark this answers is not in the document]
Comment (dmitriy-sobolev) [reply documentation/release_notes.rst:33]: Here is one more suggestion: openMP -> OpenMP.

**Review by akukanov — DISMISSED** [review]

**Review by akukanov — APPROVED** [review]

**Pull request merged**